pkg/controllers/cluster/util.go

```diff
@@ -169,22 +213,47 @@ func reconcileClusterRoleBindingInRemoteCluster(ctx context.Context, k8sClient c
 type tokenHelper struct {
 	client.Client
 	Proxy                              string
+	tokenInfo                          Claims
```

Comment:
This field is filled and accessed only in `ReconcileServiceAccountToken(..)` Could we use a local variable instead if keeping the state is not required?

Reply:
sure, will do that